views/confirmation-page.njk

```diff
@@ -0,0 +1,31 @@
+{% extends './layout.njk' %}
+
+{% block content %}
+    {% set mainClasses = "panel-background" %}
+
+    {% from "govuk/components/panel/macro.njk" import govukPanel %}
+                {{ govukPanel({
+                    titleText: "Your appeal application \n has been sent.",
+                    classes: mainClasses
+                }) }}
+                <h2 class="govuk-heading-m">What happens next</h2>
+                <ul class="govuk-list govuk-list--bullet">
+                    <li>You will receive a notification to confirm that the Tribunal has received your appeal. </li>
+                    <li>A tribunal case worker will contact you to by {{ date }} to tell you what to do next</li>
+                    <li>If you havent heard from us by {{ date }} call us on <b>0300 123 1711</b> or email customer.service@justice.gov.uk</li>
+                </ul>
+
+                <div class="testing-class">
```

Comment:
Why the testing-class? Also 'Things you can do now' block have a grey background and some padding in the design